### ISSUE TITLE
Feature: Add an option to bypass shell wrapper

### ISSUE THREAD

**Issue by r-darwish** [gerardog/gsudo#80]
## Issue Description
Due to gsudo's shell detection, trying to run a command with the Windows Powershell from Powershell Core will cause gsudo to run Powershell Core first. It would be useful if we can have a flag to disable the shell detection.

## Steps to Reproduce
Run the following command from Powershell core:
```
gsudo --debug "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.EXE" "-NoProfile" "-Command" 'echo $PSVersiontable.psversion'
```
## Screenshots

![image](https://user-images.githubusercontent.com/1383193/109123243-46931500-7752-11eb-8e92-b6bca2e4c6dc.png)

## Context:
 - Windows version: 20H2
 - gsudo version:      0.7.3

**Comment by gerardog** [comment]
Hi! Thanks for your really accurate bug report.

The shell detection is adding an extra 'powershell core' process, and the indirection makes it a quotation hell syntax.

Let me give you a shamefull workaround. Try:

```
cmd /c gsudo --debug '"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -Command "Echo $PSVersionTable.PSVersion"'
```
or without the cmd, you need to escape the `$` sign.
```
gsudo --debug `"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe`" -Command '"echo `$PSVersionTable.PSVersion"' 
```
or an example with variable substitutions:

```
$command='Echo $PSVersionTable.PSVersion'
gsudo --debug "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -Command `"($Command.Replace("$","``$"))`"
```

> It would be useful if we can have a flag to disable the shell detection.

I agree. It would imply that the command to run be something that can be executed by `CreateProcess` such as an `.EXE`, `.CMD`, `.BAT` or `.COM`. Shell statements of course won't be supported (e.g. `md` on cmd, CmdLets on PS). 

I'm unsure on how to name the 'feature', or what letter to assign to it (as in `-x`). Suggestions accepted.

**Comment by r-darwish** [comment]
Hi @gerardog, thanks for the workaround. I applied it [here](https://github.com/r-darwish/topgrade/commit/23d9a5bf63c9c45a863b2274aa819cbeccbf18c7) and it seems to work. Since this is a pretty isoteric feature that I'm asking, I think it's okay that it will only have a full named flag, such as `--no-shell`

**Comment by gerardog** [comment]
Hi @r-darwish, this will be released in v1.0. An early draft [pre-release](https://github.com/gerardog/gsudo/releases/tag/v1.0.0) is open for testing in case you want to try it. 
(please don't ship it into topgrade until released)

**Comment by gerardog** [comment]
Released in [v1.0.0](https://github.com/gerardog/gsudo/releases/tag/v1.0.0) (PR #94)
